spec/controllers/conferences_controller_spec.rb

```diff
@@ -26,6 +26,13 @@
     end
   end
 
+  describe 'GET #current' do
+    it 'redirects to first conference splashpage' do
+      current = Conference.first
```

Comment:
ok should i directly write
`expect(response).to redirect_to(conference_path(Conference.first.short_title))`